server-spi-private/src/main/java/org/keycloak/models/utils/RepresentationToModel.java

```diff
@@ -165,6 +165,9 @@ public static void importRealm(KeycloakSession session, RealmRepresentation rep,
         if (rep.getRevokeRefreshToken() != null) newRealm.setRevokeRefreshToken(rep.getRevokeRefreshToken());
         else newRealm.setRevokeRefreshToken(false);
 
+        if (rep.getRefreshTokenMaxReuse() != null) newRealm.setRefreshTokenMaxReuse(rep.getRefreshTokenMaxReuse());
+        else newRealm.setRefreshTokenMaxReuse(0);
```

Comment:
@mposolda, as discussed, default database value is not picked (at least by MySQL) for existing data as the column was set to NULLABLE to be in line with other setting columns. The default value is picked from here and it keeps the current behaviour of revoking a refresh token after first use.